ui/src/components/shared/Explainer.tsx

```diff
@@ -1,21 +1,28 @@
 import { Button, Card, Grid, Image, Space, Typography } from "antd";
 
+import { useLocalStorage } from "src/hooks/useLocalStorage";
+import { kebabCase } from "src/utils/string";
+
 export function Explainer({
   CTA,
   description,
-  handleDismiss,
   image,
   title,
 }: {
   CTA: { label: string; url: string };
   description: string;
-  handleDismiss: () => void;
   image: string;
   title: string;
 }) {
+  const [isShowing, setShowing] = useLocalStorage(`explainer-${kebabCase(title)}`, true);
```

Comment:
I suppose I am not expecting the user to intentionally break their own application by mangling its localStorage. But it's true, better to throw an error rather than having the app crash.

Reply:
Amended as discussed.